### DISCUSSION TITLE
navigationview how set item width?

### DISCUSSION THREAD

**Discussion by gatspy** [amwx/FluentAvalonia#432] [Q&A]
<img width="468" alt="image" src="https://github.com/amwx/FluentAvalonia/assets/804446/0c67a6fb-0c3a-4bd1-a4cd-3db227b221c2">

**Comment by amwx** [comment]
NavigationViewItems are automatically sized to the `OpenPaneLength` of the NavigationView's pane. What is it you're trying to do, and what have you tried that isn't working?
